apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-note.presenter.tsx

```diff
@@ -1,15 +1,28 @@
+import styled from '@emotion/styled';
+import { TextSnippet as TextSnippetIcon } from '@mui/icons-material';
+import { Box, Grid, Typography } from '@mui/material';
 import { SelfConnectionNote } from '../../../notes/hooks/use-loadable-self-notes-for-resource';
 
+const Item = styled(Box)(({ theme }) => ({
+    padding: 0,
```

Comment:
I would like to have used padding from the theme using `padding: theme.spacing(1, 0)` but I get the error that spacing does not exist:

https://mui.com/material-ui/migration/troubleshooting/#types-property-quot-palette-quot-quot-spacing-quot-does-not-exist-on-type-defaulttheme

I had trouble trying out this fix because it said `Invalid module name in augmentation, module '@mui/styles/defaultTheme' cannot be found.`

Reply:
I'd need to pair with you to help.

Reply:
Added this story: https://www.pivotaltracker.com/story/show/185592121